app/response_processor.py

```diff
@@ -128,38 +128,52 @@ def make_receipt(self, decrypted_json):
 
         return receipt_json
 
+    def _requires_downstream_processing(self, decrypted_json):
+        if 'invalid' in decrypted_json:
+            self.logger.error("Invalid survey response received, skipping notification")
+            return False
+        elif self._is_feedback_survey(decrypted_json):
+            self.logger.info("Feedback survey, skipping notification")
+            return False
+        elif decrypted_json.get("survey_id") == "lms":
```

Comment:
This might be better in an array if we need to extend it in the future, stop the building up of elif. Then we just change the log message to include it. Would make the ones we skip a bit more visible aswell

```
surveys_not_to_process = [lms, census]
...
elif decrypted_json.get("survey_id") in surveys_not_to_process
     self.logger.info('{} survey, skipping notification'.format(decrypted_json.get("survey_id"))
```

Reply:
Good point but Andrew and I discussed this and we'll skip processing based on the data version rather than survey id.